tests/Integration/Controller/ListProductsActionTest.php

```diff
@@ -18,10 +18,6 @@ public function setUp(): void
         parent::setUp();
 
         $this->mockDefaultPimAPIResponses();
-        $this->mockPimAPIResponse(
-            'get-products-scanners.json',
```

Comment:
If this was the only place where we using this file, you can also remove it :)

Reply:
I also think that a few other files are not needed anymore, like `get-products-empty-list.json`.
To easily check what can be removed, comment them in `tests/Integration/MockPimApiTrait.php` and check which tests are failing.